Documentation/Integration.md

```diff
@@ -2,29 +2,106 @@
 
 This guide covers the following options for integrating your service, with the primary use case being the integration of a new transfer service.
 
-* Integrate a new transfer service
-* Integrate a new data type
-* Integrate a new cloud provider
+* [Integrate a new transfer service](#integrate-a-new-transfer-service)
+* [Integrate a new data model](#integrate-a-new-data-model)
+* [Integrate a new cloud provider](#integrate-a-new-cloud-provider)
 
 ## Integrate a new transfer service
-A new service can be integrated into the Data Transfer Project by creating a 1) Transfer Extension and 2) Auth Extension.
- 
+A new service can be integrated into the Data Transfer Project by creating the following extensions:
+
+1. Transfer Extension
+1. Auth Extension
+1. Optionally, a new Data Model may be required if your does not exist. Please see Integrate a new Data Model (below) for more information.
+
 ### Transfer Extension
-* Transfer Extensions are located in the `extensions/data-transfer` [module](https://github.com/google/data-transfer-project/tree/master/extensions/data-transfer), where they are organized by service provider and then by data type.
-* _TODO: complete this documentation_
+
+Transfer Extensions are located in the extensions/data-transfer module, where they are organized by service provider and then by data type.
+
+1. Create a new directory for your provider under extensions/data-transfer
+
+  `$mkdir extensions/data-transfer/portability-data-transfer-foo`
```

Comment:
nit: I'm not sure if this works (or if it even matters), but I think if you add one more space before the backtick to make it line up with the "C" from create, it will indent the block with the numbered list. 
Right now it looks a bit weird (also notice how the bullet point lines up with the 3). 
![image](https://user-images.githubusercontent.com/2763488/38842646-bea8efea-41b0-11e8-8b0d-97afe90cb7a8.png)


It also might be because this is being interpreted as different lists? (a quick test for this is to make them all 1's and they should be automatically numbered correctly in the markdown view)

Reply:
(same thing regarding the bullet points between the list under Auth Extension